docs/root/intro/version_history.rst

```diff
@@ -35,6 +35,7 @@ Version history
   restart Envoy. The behavior will not switch until the connection pools are recreated. The new
   circuit breaker behavior is described :ref:`here <arch_overview_circuit_break>`.
 * upstream: changed load distribution algorithm when all priorities enter :ref:`panic mode<arch_overview_load_balancing_panic_threshold>`.
+* The stat emitted for queries without a max time set in the :ref:`MongoDB filter <config_network_filters_mongo_proxy>` was modified to emit correctly for Mongo v3.2+.
```

Comment:
nit: start with `mongo: ` and alpha sort.

Reply:
fixed; ty!